gulpfile.js

```diff
@@ -115,18 +115,19 @@ function style() {
 }
 
 function watch() {
-  style()
-
+  // Setup browserSync to reload on changes
   browserSync.init({
     server: {
       baseDir: './',
     },
     startPath: 'index.html',
-  })
+  });
+
+  // Run all required tasks once
+  style();
 
-  gulp.watch(paths.srcDir, style)
-  gulp.watch([paths.srcDir, paths.docsDir], browserSync.reload)
+  // Watch files for changes to reload when detected
+  gulp.watch([paths.docsDir, paths.styles.watch]).on('change', gulp.series(style, browserSync.reload));
 }
 
-module.exports.style = style
```

Comment:
I did a cursory search and didn't see this being used anywhere but I'm now seeing the deployment tests failing. If this was needed somewhere I didn't notice, let me know and I'll put it back.

Reply:
@kognise added it back, thanks for the feedback.